examples/apps/ai-collab/package.json

```diff
@@ -0,0 +1,112 @@
+	"scripts": {
+		"build": "fluid-build . --task build",
+		"build:compile": "fluid-build . --task compile",
+		"build:esnext": "tsc --project ./tsconfig.json && copyfiles -f ../../../common/build/build-common/src/esm/package.json ./lib",
+		"check:biome": "biome check .",
+		"check:format": "npm run check:biome",
+		"check:prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
+		"clean": "rimraf --glob dist lib next-env.d.ts \"**/*.tsbuildinfo\" \"**/*.build.log\" _api-extractor-temp nyc",
+		"eslint": "eslint --format stylish src",
+		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+		"format": "npm run format:biome",
+		"format:biome": "biome check . --write",
+		"format:prettier": "prettier --write . --cache --ignore-path ../../../.prettierignore",
+		"lint": "fluid-build . --task lint",
+		"lint:fix": "fluid-build . --task eslint:fix --task format",
+		"next:build": "next build",
+		"next:dev": "next dev",
+		"next:start": "next start",
+		"start:server": "tinylicious"
+	},
+	"c8": {
+		"all": true,
+		"cache-dir": "nyc/.cache",
+		"exclude": [
+			"src/test/**/*.*ts",
+			"dist/test/**/*.*js"
+		],
+		"exclude-after-remap": false,
+		"include": [
+			"src/**/*.*ts",
+			"dist/**/*.*js"
+		],
+		"report-dir": "nyc/report",
+		"reporter": [
+			"cobertura",
+			"html",
+			"text"
+		],
+		"temp-directory": "nyc/.nyc_output"
+	},
+	"dependencies": {
+		"@azure/identity": "^4.4.1",
+		"@azure/msal-browser": "^3.25.0",
+		"@emotion/react": "^11.13.3",
+		"@emotion/styled": "^11.13.0",
+		"@fluid-experimental/ai-collab": "workspace:~",
+		"@fluidframework/fluid-static": "workspace:~",
+		"@fluidframework/odsp-client": "workspace:~",
+		"@fluidframework/tinylicious-client": "workspace:^",
+		"@fluidframework/tree": "workspace:~",
+		"@iconify/react": "^5.0.2",
+		"@microsoft/microsoft-graph-client": "^3.0.7",
+		"@mui/lab": "6.0.0-beta.9",
+		"@mui/material": "^6.0.2",
+		"fluid-framework": "workspace:^",
+		"mui": "^0.0.1",
+		"next": "^14.2.8",
+		"notistack": "^3.0.1",
+		"react": "^18.3.1",
+		"react-dom": "^18.3.1",
+		"typechat": "^0.1.1",
+		"uuid": "^9.0.0"
+	},
+	"devDependencies": {
+		"@arethetypeswrong/cli": "^0.15.2",
+		"@biomejs/biome": "~1.8.3",
+		"@fluid-internal/mocha-test-setup": "workspace:~",
+		"@fluid-tools/build-cli": "^0.47.0",
+		"@fluidframework/build-common": "^2.0.3",
+		"@fluidframework/build-tools": "^0.47.0",
+		"@fluidframework/devtools": "workspace:^",
+		"@fluidframework/eslint-config-fluid": "^5.4.0",
+		"@microsoft/api-extractor": "7.47.8",
+		"@microsoft/microsoft-graph-types": "^2.40.0",
+		"@types/node": "^18.19.0",
+		"@types/react": "^18.3.11",
+		"@types/react-dom": "^18.3.0",
+		"@types/uuid": "^9.0.2",
+		"c8": "^8.0.1",
+		"concurrently": "^8.2.1",
+		"copyfiles": "^2.4.1",
+		"cross-env": "^7.0.3",
+		"eslint": "~8.55.0",
+		"eslint-config-prettier": "~9.0.0",
+		"moment": "^2.21.0",
+		"prettier": "~3.0.3",
```

Comment:
I expect that at least `@arethetypeswrong/cli`, `@microsoft/api-extractor`, and (optionally) `prettier` can be removed. Looks like `@types/node` probably also doesn't need included.